Library/Formula/curlish.rb

```diff
@@ -0,0 +1,15 @@
+class Curlish < Formula
+  homepage "https://pythonhosted.org/curlish/"
+  url "https://github.com/fireteam/curlish/archive/1.22.tar.gz"
+  sha1 "45e9a7d92b5f70adf257cda3f9f0f205eef0245b"
+
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
Yup, only needed in that case. Could you add a wee comment to note about `argparse`? Thanks!